atoMEC/unitconv.py

```diff
@@ -4,3 +4,4 @@
 ev_to_ha = 3.6749308136649e-2
 K_to_ha = 3.166808534191e-6
 angstrom_to_bohr = 1.8897259886
+ha_to_gpa = 1e-9 * 4.3597e-18 / ((5.2918e-11) ** 3)
```

Comment:
This would be best written as just a single number, whatever the above is equal to